requirements/software-requirements.md

```diff
@@ -77,32 +80,53 @@ This is a corollary of [Infrastructure Requirement 4](infrastructure-requirement
+software library in a particular programming language) or the software targets
+a very specific distribution/OS, then it *MUST* be packaged for a package manager
 fitting the language or OS. This should ensure the software *and all of its
 dependencies* are installable through *one single command*:
 
-* Python software *SHOULD* be packaged for and submitted to the [Python Package Index](https://pypi.org), installable through ``pip``.
-* R software should be packaged for and submitted to [CRAN](https://cran.r-project.org/).
-* Perl software *SHOULD* be packaged for and submitted to [CPAN](https://www.perl.org/cpan.html).
+* Python libraries and command-line tools *SHOULD* be packaged for and submitted to the [Python Package Index](https://pypi.org), installable through ``pip``.
+* R libraries should be packaged for and submitted to [CRAN](https://cran.r-project.org/).
+* Perl libraries *SHOULD* be packaged for and submitted to [CPAN](https://www.perl.org/cpan.html).
 * Java libraries *SHOULD* be packaged for and submitted to Maven Central, installable through ``mvn``.
-* Rust software *SHOULD* be packaged for and submitted to [crates.io](https://crates.io/), installable through ``cargo``.
-* NodeJS software *SHOULD* be packaged for and submitted to [npm](https://www.npmjs.com/), installable through ``npm``.
+* Rust libraries and command-line tools *SHOULD* be packaged for and submitted to [crates.io](https://crates.io/), installable through ``cargo``.
+* NodeJS libraries *SHOULD* be packaged for and submitted to [npm](https://www.npmjs.com/), installable through ``npm``.
 * C/C++ software has no specific ecosystem for packaging. The use of
-  distribution-specific packages is *RECOMMENDED*, targetting specific Linux
-  distributions. The use of a standardized build system such as the autotools,
+  distribution-specific packages as described further below is then *RECOMMENDED*
+  The use of an standardized build system such as the autotools,
   cmake, or make is *RECOMMENDED*. Static linking *MAY* be an appropriate solution
-  to handle dependecies.
-* Android software *SHOULD* be submitted to [Google Play](https://play.google.com) and alternatively also [F-Droid](https://f-droid.org/).
-* iOS sofware *SHOULD* be submitted to the App Store.
+  to handle dependencies.
+* (this list is not exhaustive)
 
 This step may combine both the building of software (compilation) as well the
-installation. This rule shall only be deviated from if there is no suitable
+installation. If any building/compilation is performed, then the build process
+*MUST* make use of an *automated* and *standardized* build system. 
+
+Packaging for a package manager shall only be deviated from if there is no suitable
 packaging ecosystem for the software (C/C++) or if the software is not a fit
 for the packaging ecosystem, such as the software being too high-level (e.g.  a
 complex web application) and consisting of too many components for any
```

Comment:
Not necessarily complex, though. Why would you want to publish a web application as a package? (Web) _applications_ are typically published as OCI images, only the reusable _libraries_ that they depend on should be published as packages.

Reply:
Right, perhaps is still worded a bit too strongly and we should removed the adjective *complex* here and make clear that web applications are exempted to prevent confusion.

(Technically, there is still also a place for web applications in Linux distributions of course, for example, Alpine Linux packages things like nextcloud, but I think for CLARIAH that would be overkill and may conflict with the container model which we require)

Reply:
Should be resolved with https://github.com/CLARIAH/clariah-plus/commit/3c79189412b4b4400182337aaead0a0b696564c3 now